### PR TITLE
Adjust folder structure

### PR DESCRIPTION
**Pull request by zhiyuanliang-ms**
## Why this PR?

The repo folder structure has been changed to accommodate more packages in the future.

### CODE REVIEW

File: scripts/build-and-pack.sh [hunk 1]
```diff
@@ -9,19 +9,29 @@ SCRIPT_DIR=$(dirname $(readlink -f $0))
...
-npm run test
-
-# Create a tarball.
-echo "npm pack"
-npm pack
+# Define the SDK directory.
+SDK_DIR="$PROJECT_BASE_DIR/sdk"
+
+# Loop through each package in the sdk folder.
+for PACKAGE_DIR in "$SDK_DIR"/*; do
```
Comment (Eskibear) [inline scripts/build-and-pack.sh:16]: this script is all-in-one. Suppose you will have multiple packages later, do you want to create RC bits and release for all of them everytime? If so, that's fine. If you will release one of these packages alone, I would suggest you accept package_dir as a parameter of this script.

File: sdk/feature-management/src/featureManager.ts
Comment (Eskibear) [inline sdk/feature-management/src/featureManager.ts:1]: It indeed takes me a while to find where telemetry support is added, mixed with the file renaming. A general suggestion is to make the file moving into a separate PR and focus on telemetry part in this PR. That would make it easier to review.

Comment (Eskibear) [reply sdk/feature-management/src/featureManager.ts:1]: ...and I have to double check whether there's additional changes in each renamed file.

Comment (zhiyuanliang-ms) [reply sdk/feature-management/src/featureManager.ts:1]: Revert the change. Only keep the change of folder structure change in this PR.

**Review by Eskibear — APPROVED** [review]
